tests/test_modeling_tf_auto.py

```diff
@@ -18,7 +18,13 @@
 import unittest
 
 from transformers import is_tf_available
-from transformers.testing_utils import DUMMY_UNKWOWN_IDENTIFIER, SMALL_MODEL_IDENTIFIER, require_tf, slow
+from transformers.testing_utils import (
+    DUMMY_UNKWOWN_IDENTIFIER,
```

Comment:
Typo here "UNKWOWN" -> "UNKNOWN"

Reply:
This Typo is not from this PR. 
https://github.com/huggingface/transformers/blob/a2dec768a27ab7520d4ae4f5f72643043f305fd3/src/transformers/testing_utils.py#L58
`DUMMY_UNKWOWN_IDENTIFIER`, This is used in a lot of places. Will do another PR and Fix everywhere

Reply:
https://github.com/huggingface/transformers/pull/13675

Reply:
Nice!